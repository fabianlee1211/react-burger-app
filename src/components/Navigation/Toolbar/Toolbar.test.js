import React from 'react';
import { shallow } from 'enzyme';
import Toolbar from './Toolbar';

describe('<Toolbar />', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Toolbar />);
  });

  it('renders Toolbar correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('contains 3 children node', () => {
    expect(wrapper.children()).toHaveLength(3);
  });

  it('renders a Logo and DrawerToggle', () => {
    expect(wrapper.find('DrawerToggle')).toHaveLength(1);
    expect(wrapper.find('Logo')).toHaveLength(1);
  });
});
