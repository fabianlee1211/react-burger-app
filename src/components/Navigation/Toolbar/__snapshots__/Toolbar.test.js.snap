// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Toolbar /> renders Toolbar correctly 1`] = `
<header
  className="Toolbar"
>
  <DrawerToggle />
  <div
    className="Logo"
  >
    <Logo />
  </div>
  <nav
    className="DesktopOnly"
  >
    <NavigationItems />
  </nav>
</header>
`;
